packages/react/.storybook/preview.tsx

```diff
@@ -0,0 +1,76 @@
+import '../src/icons.scss';
+import '../src/index.scss';
+import '../src/fonts.scss';
+import { Preview } from '@storybook/react';
+import { DocsContainer } from '@storybook/blocks';
+
+import { CunninghamProvider } from ':/components/Provider';
+import { COLOR_TO_THEME, getThemeFromGlobals, themes } from './themes';
+
+export const DocsWithTheme = (props, context) => {
+    const theme = getThemeFromGlobals(props.context.store.globals.globals);
+    return <CunninghamProvider theme={theme}>
+        <DocsContainer {...props} theme={themes[theme]} />
+    </CunninghamProvider>;
+};
+
+const preview: Preview = {
```

Comment:
I had to change `preview.ts` to `preview.tsx` that why everything is shown as "changed".

Reply:
I take notes for the next time